File: python/example/sort/inheritance/dog.py
from otterop.lang.string import String
from example.sort.inheritance.animal import Animal

class Dog(Animal):
    
    def __init__(self):
        super().__init__(String.wrap("I don't know"))
        self._dog_word = String.wrap("bark")

    def act(self):
        super().act()
        return self._dog_word

File: src/main/java/example/sort/inheritance/Dog.java
package example.sort.inheritance;

import otterop.lang.String;

public class Dog extends Animal {
    private String dogWord;

    public Dog() {
        super(String.wrap("I don't know"));
        this.dogWord = String.wrap("bark");
    }

    public String act() {
        super.act();
        return this.dogWord;
    }
}
